src/index.html

```diff
@@ -8,6 +8,38 @@
     <link rel="stylesheet" href="./styles/index.scss">
   </head>
   <body>
-    <h1>Product cards</h1>
+    <div class="card_container" data-qa="card">
+      <!-- PRODUCT INFO -->
+      <div class="card_product-image"></div>
+      <h1 class="card_product-name">APPLE A1419 iMac 27" Retina 5K Monoblock (MNED2UA/A)</h1>
```

Comment:
it's better to use `h2` tag since `h1` we use for page title
```suggestion
      <h2 class="card_product-name">APPLE A1419 iMac 27" Retina 5K Monoblock (MNED2UA/A)</h2>
```